docs/modding/suitmodding/addinglocals.md

```diff
@@ -0,0 +1,186 @@
+     ```json
+     "$schema": "https://raw.githubusercontent.com/atampy25/simple-mod-framework/main/Mod%20Manager/src/lib/manifest-schema.json"
+     ```
+   - If you’re using GlacierKit, it will handle much of the code suggestions and format checks.
+
+> **Note:** It might seem overkill to install Visual Studio Code for this tutorial, but you’ll appreciate the cleaner interface and error-checking features as you continue modding.
+
+---
+
+## Overview of the Process
+
+Here’s a simple summary of what we’ll be doing:
+
+1. **Identify the suit** you want to modify by locating the appropriate LOCR file.
+2. **Find the string** representing the suit's name and description.
+3. **Convert the string** into a decimal number using GlacierKit.
+4. **Modify the manifest** by adding a localisation override for the new name and description.
+5. **Save and deploy** your changes.
+
+---
+
+## Step 1: Find the LOCR File for Your Suit
+
+Most suit names and descriptions are stored in **LOCR files**. These files hold all the text for different items in the game, including reward suits.
+
+Each season has its own LOCR file, and we’ve already narrowed down the ones you’ll need:
+- **Season 3 Suits**: `004B8C5124A49543`
+- **Season 2 Suits**: `009F430D046716BE`
+- **Season 1 Suits**: `00985A1100E5EDDC`
+
+> **Tip**: If you’re not sure which season your suit belongs to, either make an educated guess or do a quick online search.
+
+### Finding the Suit
+
+1. Open **GlacierKit** and click the **"Open Project"** button.
+2. Navigate to your mod folder inside the **SMF folder** in your game’s contents.
+3. Use the **"Game Contents"** tab to paste one of the hashes above in the search bar. This will bring up the relevant LOCR file. For example, if you’re modifying a suit from Season 1, use the hash `00985A1100E5EDDC`.
+4. When the search completes, you’ll see a file tree ending in a file with a **language icon** (e.g., `randomwords.sweetmenutext].pc_localized-textlist`). This is your LOCR file.
+
+---
+
+## Step 2: Extracting the Suit’s Name and Description
+
+Now that you have the LOCR file, you need to find the specific string representing the suit you want to modify.
+
+### Finding the Suit in the LOCR File
+
+1. **Open** the LOCR file in GlacierKit, and a **preview text box** will appear on the right side.
+2. Scroll through the text or use `Ctrl + F` to search for the suit’s name (for example, "Blood Money Suit").
+   
+   You will see the suit's name appear multiple times in different languages, but the important part is the **string** associated with the name, which will look something like this:
+   ```json
+   "989928F2-06D6-42F3-871A-353F07DEF969_LEGACY_HERO_BLOODMONEYSUIT_M_HPA2293_NAME_": "Blood Money Suit"
+   ```
+   The string before the `:` is the same for each language. **Copy** this string (without the quotation marks), as it will be needed for the next step.
+
+---
+
+## Step 3: Convert the String to a Decimal Number
```

Comment:
Hash is a more accurate term than number

Reply:
Ill stick to using "strings" for now just to clearly differentiate the locr hash and the hash inside the locr that needs to be converted by calling it a "string". Itll make sense if you read the full article. I will change it later if needed